package sec03_throwsexception.EX01_ThrowsException_1;
/*
 * 작성일:2024-06-19
 * 작성자:황석현
 * 개요: 교재p487
 * 		직접 예외처리를 할 떄와 호출메서드로 예외를 전가할 때
 * 
 * */
public class ThrowsException_1 {

	public static void main(String[] args) {

	}

}
